library/src/main/java/com/getstream/sdk/chat/adapter/viewholder/attachment/AttachmentViewHolderMedia.kt

```diff
@@ -5,7 +5,8 @@ import android.view.View
 import android.view.ViewGroup
 import androidx.core.content.ContextCompat
 import androidx.core.view.isVisible
-import com.bumptech.glide.Glide
+import coil.api.load
```

Comment:
Because we have a wrapper (ImageLoader) over `Coil`, all "Coil Stuff" should be don't inside of it. Then, if we would need to switch to another alternative, it is easy to do, only one part of our code